syft/pkg/linux_kernel_package_metadata.go

```diff
@@ -0,0 +1,53 @@
+package pkg
+
+import (
+	"sort"
+
+	"github.com/scylladb/go-set/strset"
+)
+
+// LinuxKernelPackageMetadata represents all captured data for a Linux kernel
+type LinuxKernelPackageMetadata struct {
+	Name            string                      `mapstructure:"name" json:"name" cyclonedx:"name"`
```

Comment:
are we ever expecting a name here? So far I've only found blank values here (I haven't looked for that long though)

Reply:
It is theoretically possible, IIRC.